main.js

```diff
@@ -1,9 +1,11 @@
 class Car {
-  constructor($img, speed, direction, location) {
+  constructor($img, speed, direction, location, started, interval) {
```

Comment:
You can omit these `started` and `interval` parameters. Passing `true` during construction would not start the car and the `interval` is not something the car can use. If you like, you can explicitly set them to `false` and `null` on `this`, but the parameters aren't needed.